### PR TITLE
[Web analytics] Clarified Get started

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses [PCX-3973](https://jira.cfops.it/browse/PCX-3973)

### CODE REVIEW

File: content/analytics/web-analytics/getting-started/_index.md [hunk 1]
```diff
@@ -10,31 +10,31 @@ meta:
...
 
-1. Go to [Get started with Web Analytics](https://dash.cloudflare.com/sign-up/web-analytics).
-2. Enter an Email address and password.
-3. Click **Create Account**.
-4. Enter your website’s hostname.
-5. Copy the JS snippet.
-6. Add the JS snippet to any of your website’s HTML pages before the ending body tag.
-7. Click **Done**.
-8. Wait for the redirect to the Web Analytics page or click **View Analytics**. It may take a few minutes for Web Analytics data to appear.
+1. Go to [Get started with Web Analytics](https://dash.cloudflare.com/sign-up/web-analytics). Log in if you already have a Cloudflare account. If not, create one here.
```
Comment (deadlypants1973) [inline content/analytics/web-analytics/getting-started/_index.md]: is this supposed to have a link after 'here'?

File: content/analytics/web-analytics/getting-started/_index.md [hunk 2]
```diff
@@ -10,31 +10,31 @@ meta:
...
+4. Click the message box that appears to choose the hostname you have input and click **Done**.
+5. Copy the JS snippet from the **Manage site** page. This is also where you can later edit the hostname you have just added.
+6. (Optional) **Click View Analytics sites** to go back on the Web Analytics interface. If you prefer to continue setting up Web Analytics website, continue reading.
+7. Add the JS snippet to any of your website’s HTML pages before the ending body tag.
 
-Repeat steps 4-8 for all of the websites you want to track with Web Analytics by going to **Quick Actions** > **Add a site** from the Web Analytics Sites page. From this page you can also click **Manage site** inside each website's card to adjust Web Analytics for your site at any time.
+Web analytics is now set up on your website, but it may take a few minutes for Web Analytics data to appear.
+
+Repeat steps 3-7 for all the websites you want to track with Web Analytics by going to **Quick Actions** > **Add a site** from the Web Analytics Sites page. From this page you can also click **Manage site** inside each website's card to adjust Web Analytics for your site at any time.
```
Comment (deadlypants1973) [inline content/analytics/web-analytics/getting-started/_index.md]: i always write the page name without adding 'page' at the end. so for example, "from **Web Analytics Sites**."

Comment (deadlypants1973) [inline content/analytics/web-analytics/getting-started/_index.md]:
```suggestion
Repeat steps 3-7 for all the websites you want to track with Web Analytics by going to **Quick Actions** > **Add a site** from the Web Analytics Sites page. In **Web Analytics Sites**, click **Manage site** inside each website's card to adjust Web Analytics for your site at any time.
```

File: content/analytics/web-analytics/getting-started/_index.md [hunk 3]
```diff
@@ -10,31 +10,31 @@ meta:
...
 
 For more information on how many sites you can track, refer to [Limits](/analytics/web-analytics/understanding-web-analytics/limits/).
 
 ## Sites proxied through Cloudflare
 
-1. Go to [Web Analytics Sites](https://dash.cloudflare.com/?to=/:account/web-analytics) from your account home page.
+1. Go to [Web Analytics](https://dash.cloudflare.com/?to=/:account/web-analytics) from your account home page.
```
Comment (deadlypants1973) [inline content/analytics/web-analytics/getting-started/_index.md]:
```suggestion
1. Go to [Web Analytics](https://dash.cloudflare.com/?to=/:account/web-analytics) from **Account Home**.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/analytics/web-analytics/getting-started/_index.md (hunk 1 above)
Comment (marciocloudflare) [reply content/analytics/web-analytics/getting-started/_index.md]: Not really. The "here" is that page. I'm gonna re-phrase it

**Pull request merged**